portfolio/src/main/java/com/google/sps/servlets/CommentsListServlet.java

```diff
@@ -31,37 +32,33 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.time.format.DateTimeFormatter;
-import java.time.LocalDateTime;  
-
+import java.time.LocalDateTime;
 
 /** Servlet responsible for listing comments. */
 @WebServlet("/comments-list")
 public class CommentsListServlet extends HttpServlet {
-  
+
   private DatastoreService datastore;
-  
+
   @Override
-  public void init(){
+  public void init() {
     datastore = DatastoreServiceFactory.getDatastoreService();
-  } 
-  
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = Integer.parseInt(request.getParameter("max")); 
+    int pageNumber = Integer.parseInt(request.getParameter("pageNumber"));
     Query query = new Query("Comment").addSort("timestampMillis", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
     List<Comment> comments = new ArrayList<>();
-    FetchOptions fetchOps = FetchOptions.Builder.withDefaults(); 
-    
-    if(maxComments != -1){
-      fetchOps = FetchOptions.Builder.withLimit(maxComments); 
-    }
-    
+    FetchOptions fetchOps = FetchOptions.Builder.withLimit(3).offset((pageNumber - 1) * 3);
```

Comment:
What's this (3)?

Reply:
(the second (3).... curse magic numbers)

Reply:
A maximum of 3 comments is displayed in each page of the comment section